### ISSUE TITLE
Don't depend on danger-js

### ISSUE THREAD

**Issue by orj** [danger/swift#261]
One of the major reasons I was interested in Danger-swift was because I _thought_ it didn't depend on Ruby or JavaScript. Unfortunately this isn't the case. Danger-swift is just a wrapper around the JavaScript tool.

Is there any intent to actually port Danger to Swift?

**Comment by orta** [comment]
Nope, not from me - a single danger implementation takes about three years to be feature complete. A sense of software purity isn’t really worth that for me, given that you get every danger js feature (which gets multiple PRs a week)  available to many languages including Swift and it is nice to have a reference language which I care about and understand the build tools.

I’ll happily support someone willing to take it on, but I wouldn’t get your hopes up

**Comment by f-meloni** [comment]
The Danger user is not going to see/use any javascript, I think to Danger-js like just an implementation detail, is a binary that is there and you can put somewhere in your repo and just forget about it, all the code you need to write is fully swift.
The benefit of using danger-js is huge though, because the contribution to it are more than 10 times the ones we have on this repo, without it I'm pretty sure we wouldn't have any gitlab or bitbucket cloud support on danger-swift
